From: Ichthyostega <deb@ichthyostega.de>
Date: Mon, 20 Jan 2020 7:13:55 +0100
Subject: Bundle-lodepng -- no downloads please

We want a debian build to use the sources provided in the archive,
and not to download sourcecode silently from further sources in the net.

As workaround, the missing content will be supplied via debian/patches

--

Index: b/Bundle/Makefile
===================================================================
--- a/Bundle/Makefile
+++ b/Bundle/Makefile
@@ -168,13 +168,18 @@ endif
 # c7bb19ad54a53f0d9917da37de7288b1aadf6d0f is Oct 23, 2017
 # d03d7df9888aafb9c7f615895c34b05acf033908 is Mar 26, 2018
 # bd8f274fb51ef18b6d26b5f106ae4bef2150622e is Jun 24, 2018
-PNGVERSION = bd8f274fb51ef18b6d26b5f106ae4bef2150622e
 
-lodepng.cpp:
-	curl -o $@ https://raw.githubusercontent.com/lvandeve/lodepng/$(PNGVERSION)/lodepng.cpp
+##PNGVERSION = bd8f274fb51ef18b6d26b5f106ae4bef2150622e
+#
+# DEB: we provide a newer version with recent bugfixes via debian/patches
 
-lodepng.h:
-	curl -o $@ https://raw.githubusercontent.com/lvandeve/lodepng/$(PNGVERSION)/lodepng.h
+# DEB: No silent downloads of sourcecode from 'somewhere in the Net' for a Debian build please!
+#
+##lodepng.cpp:
+##	curl -o $@ https://raw.githubusercontent.com/lvandeve/lodepng/$(PNGVERSION)/lodepng.cpp
+
+##lodepng.h:
+##	curl -o $@ https://raw.githubusercontent.com/lvandeve/lodepng/$(PNGVERSION)/lodepng.h
 
 $(OBJECTPATH)/lodepng.o: lodepng.cpp lodepng.h
 $(OBJECTPATH)/ReadKrita.o: ReadKrita.cpp lodepng.h
